### PR TITLE
chore: Workspace embedding first step

### PR DESCRIPTION
**Pull request by zsinnema**
* #43 
implement route system
implement iframe for NWB Explorer

### CODE REVIEW

File: applications/osb-portal/src/components/App.tsx [hunk 1]
```diff
@@ -8,7 +10,7 @@ import SentryErrorBoundary from "./sentry/SentryErrorBoundary";
 
 import { Latest } from "./latest/Latest";
 
-import { Header, Banner, WorkspaceDrawer, Workspaces, WorkspaceToolBox, ErrorDialog } from "./index";
+import { Header, Banner, WorkspaceDrawer, Workspaces, WorkspaceToolBox, ErrorDialog, NWBExplorerFrame } from "./index";
```
Comment (filippomc) [inline applications/osb-portal/src/components/App.tsx:13]: Should be generic. Depending on the type of the workspace will show NWB explorer, NEtpyne or  a bare notebook application. A name Like WorkspaceFrame would suit better.

File: applications/osb-portal/src/components/iframe/nwbexplorer.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
```
Comment (filippomc) [inline applications/osb-portal/src/components/iframe/nwbexplorer.tsx:1]: See above: NWBExplorer -> Workspace

File: applications/osb-portal/src/components/App.tsx (hunk 1 above)
Comment (zsinnema) [reply applications/osb-portal/src/components/App.tsx:13]: Understand, question: the url setup is specific for NWB Explorer and the iFrame is generic. I mean: we would like to have an iFrame component where we can pass in the url. Right?

File: applications/osb-portal/src/components/iframe/nwbexplorer.tsx (hunk 2 above)
Comment (zsinnema) [reply applications/osb-portal/src/components/iframe/nwbexplorer.tsx:1]: yes understood

**Pull request merged**